rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -105,15 +105,19 @@ mv %{buildroot}/%{gem_instdir}/config/%{gem_name}.yaml.example \
   %{buildroot}%{foreman_pluginconf_dir}/%{gem_name}.yaml
 
 #copy init scripts and sysconfigs
+install -Dp -m0644 %{buildroot}%{gem_instdir}/%{confdir}/%{service_name}.sysconfig %{buildroot}%{sysconfig_dir}/%{service_name}
 %if 0%{?rhel} == 6
 install -Dp -m0755 %{buildroot}%{gem_instdir}/%{confdir}/%{service_name}.init %{buildroot}%{_root_initddir}/%{service_name}
 %else
-install -Dp -m0644 %{buildroot}%{gem_instdir}/%{confdir}/%{service_name}.sysconfig %{buildroot}%{sysconfig_dir}/%{service_name}
 install -Dp -m0644 %{buildroot}%{gem_instdir}/%{confdir}/%{service_name}.service %{buildroot}%{_unitdir}/%{service_name}.service
 %endif
 mkdir -p %{buildroot}%{bin_dir}
 ln -sv %{gem_instdir}/bin/%{service_name} %{buildroot}%{bin_dir}/%{service_name}
 
+#link dynflow-debug.sh to be called from foreman-debug
+chmod +x %{buildroot}%{gem_instdir}/extra/dynflow-debug.sh
+ln -s %{gem_instdir}/extra/dynflow-debug.sh %{buildroot}%{foreman_dir}/script/foreman-debug.d/60-dynflow_debug
```

Comment:
This full foreman-debug.d dir should be created with `%{__mkdir_p}` first, it's likely causing the build failure.